benchmark/benchmark.js

```diff
@@ -16,6 +16,7 @@ var IMPLS = [];
 
 
 fs.readdirSync(IMPLS_DIRECTORY).sort().forEach(function (name) {
+  if (name === '.DS_Store') { return; }
```

Comment:
`fs.readdirSync` finds the file, and tries to require it on line 21.

Might want to read only directories instead.